### PR TITLE
SBX-50: Design and implement the database schema for storing email schedules

### PR DESCRIPTION
**Pull request by lgaleana-llm**
This PR introduces the database schema for storing email schedules. It includes a new `EmailSchedule` model and a test suite for the model.

### Test Plan

pytest

### CODE REVIEW

File: test_database.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import database
+from datetime import datetime
+
+@pytest.fixture
```
Comment (lgaleana) [inline test_database.py:7]: How does this work? Should we call this `session`?

**Comment by lgaleana-llm** [comment]
The `@pytest.fixture` decorator is used to mark a function as a fixture. A fixture is a setup function that provides a fixed baseline on which tests can reliably and repeatedly execute. 

In this case, the `db` fixture sets up a mock database in memory using SQLite. It creates an engine, sets up the tables, and returns a session that can be used to interact with the database. This session is then used in the `test_email_schedule` function to add a record and commit the transaction.

The `db` fixture is automatically injected into the `test_email_schedule` function by pytest because the function has an argument with the same name as the fixture. This is a feature of pytest called fixture injection.

So, to answer your question, yes, we could call this `session` instead of `db`. However, `db` is a common name for this kind of fixture because it represents the entire database, not just a session.

**Comment by lgaleana-llm** [comment]
The `@pytest.fixture` decorator is used to mark a function as a fixture. A fixture is a setup function that provides a fixed baseline on which tests can reliably and repeatedly execute. 

In this case, the `db` fixture sets up a mock database in memory, creates the `EmailSchedule` table, and returns a `Session` object that can be used to interact with the database. This setup is done before each test that uses the `db` fixture, and torn down after the test finishes.

The `Session` object is a handle to the database, providing a high-level API for querying and manipulating the database. It's called `db` here to indicate its purpose, but it could be called `session` or anything else. Would you like me to rename it to `session`?

**Comment by lgaleana** [comment]
@lgaleana-llm rename it to session

**Review by lgaleana — APPROVED** [review]

**Pull request merged**